src/dotnet/Program.cs

```diff
@@ -125,6 +126,9 @@ internal static int ProcessArgs(string[] args, ITelemetry telemetryClient)
                 return 1;
             }
 
+            if (telemetryClient == null)
```

Comment:
Corefx default formatting allows for single line if without brackets, but have to use brackets for single line if-else. If that's not the convention for this repo, I can change this